benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -508,12 +513,6 @@ msgstr ""
 msgid "enrollment.pages.index.title"
 msgstr "Connect your card"
 
-msgid "enrollment.pages.index.mst_cc.eligibility_confirmed.headline"
-msgstr "Your eligibility is confirmed! You’re nearly there."
-
-msgid "enrollment.pages.index.mst_login.eligibility_confirmed.headline"
-msgstr "We have verified your eligibility! You’re nearly there."
-
 msgid "enrollment.buttons.payment_partner"
```

Comment:
I wonder why running `makemessages` deleted these? Is there something wrong with it?

Reply:
I undid this deletion manually.

Reply:
So if the `msgid` is not referenced anywhere in the codebase, then it is deleted. Was this `msgid` renamed/removed?

Reply:
It could have also reordered it to another part of the file...

Reply:
I think it's because of the typo on line 153 of `0002_sample_data.py`

Reply:
I added the `_`s to sample data, https://github.com/cal-itp/benefits/pull/1093/commits/0a83bf1e571d6b37b94ef5582f3eff557e4845e1 here,
then reran makemessages: https://github.com/cal-itp/benefits/pull/1093/commits/e7c435693239bf4b57a86f4f5d2ed0d33c1a23cf - and the command didn't delete anything this time!

I ran `.bin/init.sh` to check the app, and the copy is all showing up.